open-metadata-resources/open-metadata-archives/open-metadata-types/src/main/java/org/odpi/openmetadata/opentypes/OpenMetadataTypesArchive2_9.java

```diff
@@ -0,0 +1,211 @@
+     *
+     * @param archiveBuilder accumulator for types
+     */
+    public OpenMetadataTypesArchive2_9(OMRSArchiveBuilder archiveBuilder)
+    {
+        this.archiveBuilder = archiveBuilder;
+
+        this.archiveHelper = new OMRSArchiveHelper(archiveBuilder,
+                                                   archiveGUID,
+                                                   originatorName,
+                                                   creationDate,
+                                                   versionNumber,
+                                                   versionName);
+    }
+
+
+    /**
+     * Return the unique identifier for this archive.
+     *
+     * @return String guid
+     */
+    public String getArchiveGUID()
+    {
+        return archiveGUID;
+    }
+
+
+    /**
+     * Returns the open metadata type archive containing all of the standard open metadata types.
+     *
+     * @return populated open metadata archive object
+     */
+    public OpenMetadataArchive getOpenMetadataArchive()
+    {
+        final String methodName = "getOpenMetadataArchive";
+
+        if (this.archiveBuilder != null)
+        {
+            /*
+             * Build the type archive.
+             */
+            this.getOriginalTypes();
+
+            /*
+             * The completed archive is ready to be packaged up and returned
+             */
+            return this.archiveBuilder.getOpenMetadataArchive();
+        }
+        else
+        {
+            /*
+             * This is a logic error since it means the creation of the archive builder threw an exception
+             * in the constructor and so this object should not be used.
+             */
+            throw new OMRSLogicErrorException(OMRSErrorCode.ARCHIVE_UNAVAILABLE.getMessageDefinition(),
+                                              this.getClass().getName(),
+                                              methodName);
+        }
+    }
+
+
+    /**
+     * Add the types from this archive to the archive builder supplied in the
+     * constructor.
+     */
+    public void getOriginalTypes()
+    {
+        OpenMetadataTypesArchive2_7  previousTypes = new OpenMetadataTypesArchive2_7(archiveBuilder);
+
+        /*
+         * Pull the types from previous releases.
+         */
+        previousTypes.getOriginalTypes();
+
+        /*
+         * Calls for new and changed types go here
+         */
+        add0335PrimaryCategoryClassification();
+
+    }
+
+
+
+    /**
+     * 0335  Add the Primary Category Classification
+     */
+    private void add0335PrimaryCategoryClassification()
+    {
+        this.archiveBuilder.addClassificationDef(addPrimaryCategoryClassification());
+    }
+
+    private ClassificationDef addPrimaryCategoryClassification()
+    {
+        final String guid            = "3a6c4ba7-3cc5-48cd-8952-bwra92da016d";
+        final String name            = "PrimaryCategoryClassification";
```

Comment:
Do not need the word Classification in the title - it should be "PrimaryClassification"